### PR TITLE
Adding teams theme

### PR DESCRIPTION
**Pull request by dzearing**
This adds a placeholder Teams customization set as an example.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/customizations/TeamsCustomizations.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { ICustomizerProps } from '../Utilities';
+import { ITheme, createTheme } from '../Styling';
+import { IButtonStyles } from '../Button';
+
+/**
+ * To consume a customization set, use the Customizer component and wrap your app:
+ *
+ *   import { Customizer } from 'office-ui-fabric-react';
+ *   import { TeamsCustomizations } from 'office-ui-fabric-react/lib/customizations/TeamsCustomizations';
```
Comment (cliffkoh) [inline packages/office-ui-fabric-react/src/customizations/TeamsCustomizations.tsx]: Is this violating the guidelines about not importing beyond top level imports?

Comment (cliffkoh) [reply packages/office-ui-fabric-react/src/customizations/TeamsCustomizations.tsx]: Specifically, we don't usually treat anything that isn't exposed at the top level to be part of API surface. Would you treat this addition as part of the package's API surface? If so, the change should be minor.. if not, this line over here violates best practices :)

**Comment by dzearing** [comment]
Let me merge this one. I spotted a bug.

File: packages/office-ui-fabric-react/src/customizations/TeamsCustomizations.tsx (hunk 1 above)
Comment (dzearing) [reply packages/office-ui-fabric-react/src/customizations/TeamsCustomizations.tsx]: I don't want to add it to top level imports, as it is example code (just like all of the examples.)

Comment (cliffkoh) [reply packages/office-ui-fabric-react/src/customizations/TeamsCustomizations.tsx]: Should we just add a single 1 line comment above the import saying this is just an example (and not part of the API surface?)

**Comment by dzearing** [comment]
I'm going to merge, as the theme itself is not complete anyway. The button styling is not quite perfect yet, but I will adjust in a separate pr.

**Pull request merged**